static/app/views/performance/browser/resources/resourceSummaryPage/sampleImages.tsx

```diff
@@ -125,9 +125,16 @@ function SampleImagesChartPanelBody(props: {
   return (
     <ImageWrapper>
       {images.map(resource => {
+        const hasRawDomain = Boolean(resource[RAW_DOMAIN]);
+        const isRelativeUrl = resource[SPAN_DESCRIPTION].startsWith('/');
+
+        const src =
+          isRelativeUrl && hasRawDomain
+            ? `${resource[RAW_DOMAIN]}${resource[SPAN_DESCRIPTION]}`
+            : resource[SPAN_DESCRIPTION];
```

Comment:
I think string concatenation is probably fine here, but it's be safer to use something like the `URL` constructor, that would make sure the trailing slashes and so on are treated correctly

Reply:
Yeah I like this idea! Will make the change